lib/widgets/movie/movie_info.dart

```diff
@@ -25,48 +27,90 @@ class MovieInfo extends StatelessWidget {
   static const double padGenres = 10;
   static const double separation = 10;
 
+  @override
+  State<MovieInfo> createState() => _MovieInfoState();
+}
+
+class _MovieInfoState extends State<MovieInfo> {
+  late final Future<List<String>> _movieGenres;
+  MyLocalRepository provider = MyLocalRepository();
```

Comment:
This should be of type `IMyRepository`. Also follow the naming convention suggested by @FranciscoGaraventa

Reply:
This refactor will happen in the next PR

Reply:
Done it (?